unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/PlacesAndEventsSectionComponentView.cs

```diff
@@ -40,18 +40,14 @@ public override void Dispose()
 
     internal void CreateSubSectionSelectorMappings()
     {
-        subSectionSelector.RefreshControl();
         subSectionSelector.GetSection(0)?.onSelect.AddListener((isOn) => placesSubSection.gameObject.SetActive(isOn));
         subSectionSelector.GetSection(1)?.onSelect.AddListener((isOn) => eventsSubSection.gameObject.SetActive(isOn));
```

Comment:
We should identified with constant or enums which section represent each number for clarify

Reply:
Done!